### DISCUSSION TITLE
New docs: Consider making the copy button cursor pointer

### DISCUSSION THREAD

**Discussion by mwala-zm** [rails/rails#51384] [Docs]
Hi there,

I noticed that the code snippets' copy button element could look better if it had the `cursor pointer` component added to the `clipboard-button` class. Currently, it looks a bit static. Apart from that, I think the documentation looks amazing and is a breath of fresh air.

@AmandaPerino

**Comment by AmandaPerino** [comment]
Hi @mwala-zm - good catch! We'll get this added to a future update.

**Comment by AmandaPerino** [comment]
This has been added. Thanks again @mwala-zm.
